packages/autorest.typescript/src/restLevelClient/transforms/transform.ts

```diff
@@ -42,7 +42,7 @@ export function transform(model: CodeModel): RLCModel {
     schemas: transformSchemas(model),
     responses: transformResponseTypes(model, importDetails),
     importSet: importDetails,
-    apiVersionParam: transformApiVersionParam(model),
+    apiVersionInQueryParam: transformApiVersionParam(model),
```

Comment:
Rename the property because it only indicates the api-version in query param. If we have api-version in url it will save in UrlInfo